common/init.sls

```diff
@@ -45,8 +45,15 @@ host-{{ hostname }}:
 {% endfor %}
 
 {% for ssh_user in common.ssh_users %}
+{{ ssh_user }}:
+  user.present:
+    - home: /home/{{ ssh_user }}
+    - optional_groups:
+        - wheel
+    - empty_password: True
+
 sshkey-{{ ssh_user }}:
   ssh_auth.present:
```

Comment:
We can combine this state with the previous one, so that they share the same ID.